ksicht/settings.py

```diff
@@ -75,6 +75,7 @@
     "imagekit",
     "ksicht.core",
     "ksicht.bulma",
+    'fontawesomefree',
```

Comment:
There is no need to add fontawesome this way as it already is included in the project. However, it is bundled with webpack. In order for this to work properly, you'll need to start project with `make run` as stated in [the README file](https://github.com/xaralis/ksicht#running-the-app). This will spawn  both Django app *and* Webpack dev server.

Reply:
I've tried to use included FA package, but it was not working for Facebook ( `fab fa-facebook` ) and Instagram ( `fab fa-instagram` ) icons - was showing empty boxes instead. I'm using `make run`.

Reply:
I think you need to add `@import "~@fortawesome/fontawesome-free/scss/brands";` [here](https://github.com/FelyCZ/ksicht/blob/update-sep23/assets/sass/main.scss#L16)

Reply:
Thanks, works now!